packages/react/src/side-nav/SideNavLink.tsx

```diff
@@ -70,7 +75,25 @@ export function SideNavLink({
 				}}
 			>
 				<SideNavLinkIndicator depth={depth} />
-				<span>{label}</span>
+
+				<span css={{ flex: 1 }}>{label}</span>
+
+				{Boolean(children) &&
+					(isOpen ? (
+						<ChevronDownIcon
+							aria-hidden={false}
+							aria-label="(related links below)"
+							css={depth > 1 ? { marginRight: mapSpacing(0.25) } : undefined}
+							size={depth > 1 ? 'sm' : 'md'}
+						/>
+					) : (
+						<ChevronRightIcon
+							aria-hidden={false}
+							aria-label="(related links available after opening)"
```

Comment:
Interesting. The wording is interesting. "Related links" while technically correct feels a bit weird as related links often means "tangential links" that you might find in an aside.

When you land on a sub list, both VoiceOver and TalkBack tell you that they are level 2 items, so "level" is a construct screen reader users understand. Perhaps we could lean in to this with something like "Has sub-level below"? The "after opening" is a bit weird, too. Perhaps something like "Sub-level visible after navigating"?>

And it would be really nice if screen readers read brackets with a pause, but they don't, so I think scrap the brackets and start them with a full stop to create a natural pause

Reply:
- [ ] change chevron aria-label to `. Has 7 nested links`, for example